formula-android-tests/src/test/java/com/instacart/formula/TestFormulaRule.kt

```diff
@@ -33,4 +51,8 @@ class TestFormulaRule(
         val context = ApplicationProvider.getApplicationContext<Application>()
         initFormula(context)
     }
+
+    private fun assertNoErrors() {
```

Comment:
Were existing tests triggering an error but we just missed it?

Reply:
Yeah, it was being swallowed.